.github/workflows/npm-publish-cht-deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish cht-deploy to npm
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./scripts/deploy
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run tests
+        run: npm test
```

Comment:
This build only runs on master, right? Do we run these tests from some other CI process? If they only run in master then we won't find out we've broken something until after merge. Consider adding this to the main build.yml so we can test before merging too.

Reply:
Yup, they are [run](https://github.com/medic/cht-core/blob/master/package.json#L53) before PRs are merged. I guess that effectively makes running the tests here redundant as tests must pass before PRs are merged anyway. I will remove it.

Reply:
Perfect!